conda/cli/conda_argparse.py

```diff
@@ -125,20 +129,26 @@ def do_call(args, parser):
     module = import_module(relative_mod, __name__.rsplit(".", 1)[0])
 
     command = relative_mod.replace(".main_", "")
-    _run_pre_command_hooks(command, args)
+    _run_command_hooks("pre", command, args)
+    result = getattr(module, func_name)(args, parser)
+    _run_command_hooks("post", command, args, result)
 
-    return getattr(module, func_name)(args, parser)
+    return result
 
 
-def _run_pre_command_hooks(command: str, args) -> None:
+def _run_command_hooks(hook_type: CommandHookTypes, *args) -> None:
```

Comment:
The command is always needed to be passed (we don't have a "catch-all pre/post command hook"), let's just keep the argument, so we don't have to unpack the args later.

```suggestion
def _run_command_hooks(hook_type: CommandHookTypes, command: str, *args, **kwargs) -> None:
```